src/explore-education-statistics-common/src/components/Accordion.tsx

```diff
@@ -108,10 +108,31 @@ const Accordion = ({
       }, 200);
     };
 
+    const openHash = () => {
```

Comment:
Yep, the existing code works. After a while of discussing it with you & looking through `AccordionSection`, If figured out that replacing any heading ID that starts with a number and giving it a value such as `section-` fixed the issue

Reply:
I don't think this is quite right - testing on http://localhost:3000/methodology/permanent-and-fixed-period-exclusions-in-england adding the id eg '#section-.-national-statistics-badging' doesn't open the section. Maybe the '.' makes the id invalid?
Looking more closely at it I see that the buttons in the accordion h2s have an id on them based on the section number, so I wonder if this could be re-used instead? It would have to have something appended or removed from it so that it's not exactly the same as the button id.

Reply:
Yeah, I've had a look at it does seem that the erroneous `.` makes the id invalid. I think that re-using the button-id with something such as a `-` to provide variation is the best option as you say. In the unlikely event that a methodology accordion has the same title, performing `.replace(/[0-9]/, 'section-'),` would end up results in non-unique IDs for different accordion sections